src/databases/Blueprint/avtBlueprintFileFormat.C

```diff
@@ -1711,6 +1712,38 @@ avtBlueprintFileFormat::GetTime()
     }
 }
 
+// ****************************************************************************
+//  Method: GenerateOffsetsForPolytopalMesh
+//
+//  Purpose:
+//      Polyhedral meshes require offsets to verify. We can generate them
+//      before verification takes place.
+//
+//  Arguments:
+//      data        The mesh domain we wish to add offsets to.
+//
+//  Programmer: Justin Privitera
+//  Creation:   10/25/23
+//
+//  Modifications:
+//
+// ****************************************************************************
+void
+GenerateOffsetsForPolytopalMesh(Node &data)
+{
+    // using short circuit evaluation, so these statements will only execute
+    // until we hit the first one that is false
+    if (data.has_child("topologies") && 
+        data["topologies"].number_of_children() > 0 && 
+        data["topologies"][0].has_child("type") && 
```

Comment:
If my file had another topology first, is this logic going to fail by only considering the first topology?

Reply:
I think that `ReadBlueprintMesh(domain, abs_meshname_str, data);` (called above this function each time) only returns a single topology.

Reply:
I have just verified that this is indeed the case.

Reply:
I can put a note in that says that we assume there will only ever be a single topo when we arrive at this logic.